cmdk/src/index.tsx

```diff
@@ -508,6 +511,10 @@ const Command = React.forwardRef<HTMLDivElement, CommandProps>((props, forwarded
               next(e)
               break
             }
+            case 'Tab': {
```

Comment:
This feels dangerously close to an accessibility violation – how would you Tab to other interactive elements within the dialog if Tab is being used for navigation?

Reply:
Thanks for reviewing the code @pacocoursey.
This PR is actually inspired by Linear's command line

https://user-images.githubusercontent.com/21135812/219544664-1d66065d-9963-4b61-bc20-12d6ccb33e50.mp4

Can I get a hint for better implementation?

Reply:
I don't think there's a better implementation, this simply isn't the right way to handle [Tab] with this type of component pattern, so I don't think we can add it.

Reply:
Alright, I understand. I'll close this PR. Thanks, Paco.